app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -480,7 +480,7 @@ public boolean onLongClick(View v) {
                             checked = i;
                         }
                         choices[i] = String.valueOf(values[i]) + " "
-                                + getString(R.string.time_unit_seconds);
+                                + getString(R.string.time_seconds);
```

Comment:
removing the number doesn't sound like a good idea.
better leave it this way, if we need time units without a number it makes sense to have a separate string